src/serverConfig-example.ts

```diff
@@ -3,23 +3,33 @@
 // Best is to Keep them as a reference
 // Just duplicate the file and name it serverConfig.ts to adjust to your likings
 
+
+// you can derive the following options by looking at the /index.html file of your jitsi-meet-web instance (i.e. the server, where you can create new sessions)
+
 export const connectionOptions = {
   hosts: {
-     domain: 'meet.jit.si',
-     muc: 'conference.meet.jit.si', 
-     focus: 'focus.meet.jit.si',
```

Comment:
Where did the focus domain go?

Reply:
don't really know. question is: what is it needed for? apparently, my setup works without one…

Reply:
This allows you to explicitly specify the focus component to use (part of the XMPP stuff) it defaults to the value of `hosts.domain` if not set.

Reply:
tried to address this in 21e3cf9.